tests/Graph/Models/Node.php

```diff
@@ -6,12 +6,18 @@
 use Illuminate\Database\Eloquent\Relations\HasMany;
 use Illuminate\Database\Eloquent\SoftDeletes;
 use Illuminate\Database\Query\Expression;
+use Illuminate\Support\Carbon;
 use Staudenmeir\EloquentHasManyDeep\HasManyDeep;
 use Staudenmeir\EloquentHasManyDeep\HasOneDeep;
 use Staudenmeir\EloquentHasManyDeep\HasRelationships;
 use Staudenmeir\EloquentHasManyDeep\HasTableAlias;
 use Staudenmeir\LaravelAdjacencyList\Eloquent\HasGraphRelationships;
 
+/**
+ * @property Carbon|null $deleted_at
+ *
+ * @property-read \Illuminate\Database\Eloquent\Relations\Pivot $pivot
```

Comment:
Not 100% happy with adding this as a docblock, since the pivot is only present when queried via a relation call, but it solves the errors. Open for other directions though

Reply:
👍 I think that's perfectly fine for test files.